sessionctx/variable/varsutil_test.go

```diff
@@ -564,9 +559,6 @@ func (s *testVarsutilSuite) TestValidate(c *C) {
 		{TiDBEnableTablePartition, "OFF", false},
 		{TiDBEnableTablePartition, "AUTO", false},
 		{TiDBEnableTablePartition, "UN", true},
-		{TiDBEnableListTablePartition, "ON", false},
-		{TiDBEnableListTablePartition, "OFF", false},
-		{TiDBEnableListTablePartition, "list", true},
```

Comment:
I actually think this should be global + session scoped, but I'll leave that to a separate discussion. Because the validation scope now tests scope, this needed fixing.